core-bundle/contao/templates/_new/component/_stylesheet.html.twig

```diff
@@ -19,7 +19,7 @@
 {% block stylesheet_component %}
     {% set stylesheet = stylesheet|default(_context) %}
 
-    {% if stylesheet.lazy|default(true) %}
+    {% if stylesheet.lazy is defined ? stylesheet.lazy : true %}
```

Comment:
I liked the shorter version more. But either way, why is the default `true`? If the style sheet is lazy, it is loaded lazily. If the style sheet does not define anything, it is loaded lazily, too?

Reply:
Yes, it defaults to true but you can pass `lazy: false` if you want to load it synchronously

Reply:
> I liked the shorter version more.

The short version doesn’t work if `false` is not the default because `false|default(true)` is still true.

Reply:
> Yes, it defaults to true but you can pass `lazy: false` if you want to load it synchronously

I would then prefer to call the option `synchronously: true` instead of lazy: false`. WDYT?

Reply:
`synchronously` would be way hard to type for non-native speakers I guess 🙈😅

Maybe `sync: true`? But I’d still prefer `lazy: false` because the term “lazy loading” is pretty well understood I think.